x/stakeibc/module_ibc.go

```diff
@@ -211,7 +186,40 @@ func (im IBCModule) NegotiateAppVersion(
 	return proposedVersion, nil
 }
 
+func (im IBCModule) UnmarshalAck(ctx sdk.Context, acknowledgement []byte) (*channeltypes.Acknowledgement_Result, error) {
+	ack := channeltypes.Acknowledgement_Result{}
+	eventType := "callback"
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			eventType,
+			sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
+			sdk.NewAttribute(types.AttributeKeyAck, fmt.Sprintf("%v", ack)),
```

Comment:
is this supposed to be emitted as bytes? I assume so but just double checking

Reply:
might not make much sense to log this to be honest, removed this line